api/src/main/resources/moduleApplicationContext.xml

```diff
@@ -35,7 +35,7 @@
 							<property name="dao">
 								<bean class="${project.parent.groupId}.${project.parent.artifactId}.api.db.hibernate.HibernateFormFilterDAO">
 									<property name="sessionFactory">
-										<ref bean="dbSessionFactory" />

```

Comment:
With the new strategy, you do not need a change in this file.

Reply:
oh ive tried to leave  it like that but it failed to build

Reply:
After reverting this change, did you commit?
Secondly, you need to squash these commits into one.

Reply:
i throws an exception if i leave the the refernece to dbSession , otherwise , i just have to do this if am to leve the dbSEssion reference there

Reply:
Try remove this change and i test it out.